### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by im-minion**
Added whl file downloading links of Python 3.6 for Linux with CPU only and GPU both :)

### CODE REVIEW

**Comment by im-minion** [comment]
I signed it.

File: README.md [hunk 1]
```diff
@@ -46,8 +46,10 @@ packages on Linux, Mac, and Windows.
...
...tf_nightly_gpu-1.head-cp27-none-linux_x86_64.whl) ([build history](https://ci.tensorflow.org/view/tf-nightly/job/tf-nightly-linux/TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON2,label=gpu-linux/)) / [Python 3.4](https://ci.tensorflow.org/view/tf-nightly/job/tf-nightly-linux/TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON3,label=gpu-linux/lastSuccessfulBuild/artifact/pip_test/whl/tf_nightly_gpu-1.head-cp34-cp34m-linux_x86_64.whl) ([build history](https://ci.tensorflow.org/view/tf-nightly/job/tf-nightly-linux/TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON3,label=gpu-linux/)) / [Python 3.5](https://ci.tensorflow.org/view/tf-nightly/job/tf-nightly-linux/TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON3.5,label=gpu-linux/lastSuccessfulBuild/artifact/pip_test/whl/tf_nightly_gpu-1.head-cp35-cp35m-linux_x86_64.whl) ([build history](https://ci.tensorflow.org/view/tf-nightly/job/tf-nightly-linux/TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON3.5,label=gpu-linux/)) / [Python 3.6](https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow_gpu-1.4.0-cp36-cp36m-linux_x86_64.whl
```
Comment (av8ramit) [inline README.md]: This is technically incorrect. These links are of the tf-nightly jobs and their whl files are found on ci.tensorflow.org under the tf-nightly tab. You have linked the official 1.4.0 release.

**Review by av8ramit — CHANGES_REQUESTED** [review]

**Comment by av8ramit** [comment]
Thanks @im-minion. You're correct in that the readme does not have updated links to 3.6 whl files.

File: README.md (hunk 1 above)
Comment (im-minion) [reply README.md]: oh yes, you are right.. @av8ramit ...will u please tell me how can I edit this PR or can accept changes done from your side.. as I am a newbie to open source contributions

Comment (av8ramit) [reply README.md]: Here are the links you want:

http://ci.tensorflow.org/view/tf-nightly/job/tf-nightly-linux/TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON3.6,label=cpu-slave/lastSuccessfulBuild/artifact/pip_test/whl/tf_nightly-1.head-cp36-cp36m-linux_x86_64.whl


http://ci.tensorflow.org/view/tf-nightly/job/tf-nightly-linux/TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON3.6,label=gpu-linux/lastSuccessfulBuild/artifact/pip_test/whl/tf_nightly_gpu-1.head-cp36-cp36m-linux_x86_64.whl

Update with those and I'll approve. No worries, thanks for contributing!

Comment (im-minion) [reply README.md]: sorry @av8ramit ..mistakenly I deleted that forked tensorflow repo from my profile ...I reforked that and created the new PR please check at
https://github.com/tensorflow/tensorflow/pull/15666
Thanks for the help @av8ramit :+1:

**Comment by av8ramit** [comment]
Closing: #15666

File: README.md (hunk 1 above)
Comment (av8ramit) [reply README.md]: No problem.

**Pull request closed without merging**